src/core/inlines.js

```diff
@@ -77,10 +77,13 @@ const inlineElement = /(?:\[\^[^^]+\^\])/; // Inline [^element^]
  */
 function inlineElementMatches(matched) {
   const value = matched.slice(2, -2).trim();
-  const [element, attribute] = value.split("/", 2).map(s => s && s.trim());
+  const [element, attribute] = value
+    .replace("\\/", "%%")
+    .split("/", 2)
+    .map(s => s && s.trim().replace("%%", "/"));
```

Comment:
In future, we can do:
``` js
const [element, attribute] = value.split(/(?<!\\)\//, 2).map(s => s && s.trim());
const textContent = element.replace("\\/", "/")
```

``` js
// example
const fn = s => s.split(/(?<!\\)\//, 2).map(s => s && s.trim().replace("\\/", "/"));
fn("application/x-www-form-urlencoded");
// Array [ "application", "x-www-form-urlencoded" ]
fn("application\\/x-www-form-urlencoded")
// Array [ "application/x-www-form-urlencoded" ]
```

Open to better ideas.

Reply:
What about just `.split(/\\?\//, 2)`?

Reply:
LGTM for now, maybe better to define this as a separate function as this is also used elsewhere. A comment about the future change will be great.